### PR TITLE
Swipes Page

### PR DESCRIPTION
**Pull request by allie-rae**
### Updated swipes page:
localhost_3000_swipe
localhost_3000_swipe (2)

### No more swipes component:
Screen Shot 2022-09-27 at 1 36 42 AM

### CODE REVIEW

File: src/theme/palette.ts [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
+export const primaryDark = 'rgb(44, 56, 126)';
+export const primaryContrast = '#fff';
+
+export const secondaryMain = '#f50057';
+export const secondaryLight = 'rgb(247, 51, 120)';
+export const secondaryDark = 'rgb(171, 0, 60)';
+export const secondaryContrast = '#fff';
+
+export const errorMain = '#f44336';
+export const errorLight = '#e57373';
```
Comment (jae-finger) [inline src/theme/palette.ts:12]: This is my favorite hex code!

File: tsconfig.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
```
Comment (jae-finger) [inline tsconfig.json:9]: ????? FALSE?????

**Review by jae-finger — APPROVED** [review]

File: src/theme/palette.ts (hunk 1 above)
Comment (allie-rae) [reply src/theme/palette.ts:12]: 🤣

File: tsconfig.json (hunk 2 above)
Comment (allie-rae) [reply tsconfig.json:9]: Your reviews are hilarious 🤣

**Pull request merged**